src/FormHandler/FormHandler.php

```diff
@@ -18,19 +24,10 @@ public function __construct(EntityManagerInterface $manager)
 
   public function handleForm(Form $form): void
   {
-    $id = $form->get('id')->getData();
+    $user = $this->manager->getRepository(User::class)->findOneBy(['email' => $form->getData()->getAuthorEmail()]);
```

Comment:
@JanMaria To trzeba zmienić. Wykorzystaj możliwości jakie daje Ci typ pola formularza `EntityType`. Jak to zrobisz to usuń pole `authorEmail` z encji.